core/chains/evm/config/toml/defaults/Hedera_Mainnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '295'
+ChainType = 'hedera'
+#considering the 3-5 (6 including a buffer) seconds of finality and 2 seconds block production
+#we set the depth to 6/2 = 3 blocks, setting to 10 for safety
+FinalityDepth = 10
+#Hedera has high TPS, so polling less often 
+LogPollInterval = '10s'
+MinIncomingConfirmations = 1
+#turning off the liveness detection, since the Websocket connection is not reliable, can be turned on if needed
+NoNewHeadsThreshold = '0'
```

Comment:
@dhaidashenko : Does this comment and setting look good?
If Hedera has a 2 second block rate, we assume heads to come regularly.
Websock not being reliable doesn't seem to imply we should set this to 0.

Also, how close are we to doing liveness checks via Polling?

Reply:
HeadTracker uses websocket connection to get new heads. If there is no new heads, TXM's confirmer won't mark txs as confirmed and will eventually get stuck. It's highly recommended to enable this check, especially if the ws connection is not reliable. It will force to create a new connection, thus we'll have a chance to auto resolve the issue.

Reply:
Since we have only 1 working RPC, we cannot mark that as out-of-sync, especially with un-reliable Websocket endpoints, we set it to zero, so it waited till auto recovered, this again will be switched back on once we get 2 more fully functioning RPC